app/views/actors/_form.html.erb

```diff
@@ -0,0 +1,32 @@
+<%= form_with(model: actor, local: true) do |form| %>
+  <%= render "shared/errors", object: actor %>
+
+  <div class="field">
+    <%= form.label :image %>
+    <%= form.file_field :image %>
+  </div>
+  
+  <div class="field">
+    <%= form.label :name %>
+    <%= form.text_field :name, autofocus: true %>
+  </div>
+
+  <div class="field">
+    <%= form.label :birth_date %>
+    <%= form.date_field :birth_date %>
+  </div>
+
+  <div class="field" id="gender">
+    <%= form.label :gender %>
+    <div>
+    <%= form.radio_button :gender, :male %> Male
```

Comment:
Can we use https://apidock.com/rails/v4.0.2/ActionView/Helpers/FormOptionsHelper/collection_radio_buttons